ios/src/TGGeoPoint.h

```diff
@@ -0,0 +1,27 @@
+//
+//  TGGeoPoint.h
+//  tangram
+//
+//  Created by Karim Naaji on 10/27/16.
+//
+//
+
+#ifndef TGGeoPoint_h
+#define TGGeoPoint_h
```

Comment:
It's just for portability. I used the `#ifndef` syntax to make it c-compliant, if someones uses `include` syntax in objective-c for c headers this would still work without using objective-c++ with modern c++ syntax. 
Also relevant: http://stackoverflow.com/questions/23696115/is-pragma-once-part-of-the-c11-standard#23699893